<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWMainWind</class>
 <widget class="QMainWindow" name="QWMainWind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QWMainWind</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTextEdit" name="txtEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>826</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_option"/>
    <addaction name="separator"/>
    <addaction name="action_close"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="action_cut"/>
    <addaction name="action_copy"/>
    <addaction name="action_paste"/>
    <addaction name="separator"/>
    <addaction name="action_clear"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>格式</string>
    </property>
    <addaction name="action_bold"/>
    <addaction name="action_italic"/>
    <addaction name="action_underline"/>
    <addaction name="separator"/>
    <addaction name="action_show_label"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_new"/>
   <addaction name="action_option"/>
   <addaction name="action_clear"/>
   <addaction name="separator"/>
   <addaction name="action_cut"/>
   <addaction name="action_copy"/>
   <addaction name="action_paste"/>
   <addaction name="separator"/>
   <addaction name="action_italic"/>
   <addaction name="action_bold"/>
   <addaction name="action_underline"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_cut">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/cut.bmp</normaloff>:/images/images/cut.bmp</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/120.bmp</normaloff>:/images/images/120.bmp</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>拷贝内容</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/paste.bmp</normaloff>:/images/images/paste.bmp</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_bold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/BLD.BMP</normaloff>:/images/images/BLD.BMP</iconset>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
   <property name="toolTip">
    <string>粗体</string>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_italic">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/ITL.BMP</normaloff>:/images/images/ITL.BMP</iconset>
   </property>
   <property name="text">
    <string>斜体</string>
   </property>
   <property name="toolTip">
    <string>斜体</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action_underline">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/UNDRLN.BMP</normaloff>:/images/images/UNDRLN.BMP</iconset>
   </property>
   <property name="text">
    <string>下划线</string>
   </property>
   <property name="toolTip">
    <string>下划线</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="action_close">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/exit.bmp</normaloff>:/images/images/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_option">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/open3.bmp</normaloff>:/images/images/open3.bmp</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开一个文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_clear">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/103.bmp</normaloff>:/images/images/103.bmp</iconset>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
   <property name="toolTip">
    <string>清空内容</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+C</string>
   </property>
  </action>
  <action name="action_set_font">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/20.ico</normaloff>:/images/images/20.ico</iconset>
   </property>
   <property name="text">
    <string>字体设置</string>
   </property>
   <property name="toolTip">
    <string>字体设置</string>
   </property>
  </action>
  <action name="action_new">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/new2.bmp</normaloff>:/images/images/new2.bmp</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建文档</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_show_label">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/430.bmp</normaloff>:/images/images/430.bmp</iconset>
   </property>
   <property name="text">
    <string>显示工具栏文字标签</string>
   </property>
   <property name="toolTip">
    <string>显示工具栏文字标签</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_cut</sender>
   <signal>triggered()</signal>
   <receiver>txtEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_paste</sender>
   <signal>triggered()</signal>
   <receiver>txtEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_copy</sender>
   <signal>triggered()</signal>
   <receiver>txtEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_close</sender>
   <signal>triggered()</signal>
   <receiver>QWMainWind</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_clear</sender>
   <signal>triggered()</signal>
   <receiver>txtEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
